src/main/java/org/jfrog/hudson/pipeline/common/executors/NpmEnvExtractor.java

```diff
@@ -29,8 +29,8 @@ public NpmEnvExtractor(Run build, BuildInfo buildInfo, Deployer deployer, Resolv
 
     @Override
     protected void addExtraConfiguration(ArtifactoryClientConfiguration configuration) {
-        configuration.npmHandler.setNpmInstallArgs(args);
-        configuration.npmHandler.setNpmPath(path);
-        configuration.npmHandler.setNpmModule(module);
+        configuration.buildToolHandler.setBuildToolArgs(args);
```

Comment:
buildToolHandler --> packageManagerHandler

setBuildToolArgs --> setArgs

Reply:
This change will be added in #314 due to a required change in the [build-info project](https://github.com/jfrog/build-info/pull/363).